lang/lua-eco/Makefile

```
@@ -82,7 +84,7 @@ define Package/lua-eco-ssl/config
 		depends on PACKAGE_libopenssl
 	config LUA_ECO_WOLFSSL
 		bool "wolfSSL"
-		depends on PACKAGE_libwolfssl
+		depends on PACKAGE_libwolfssl || PACKAGE_libwolfssl-cpu-crypto
```

It seems the package has changed to:

```suggestion
		depends on PACKAGE_libwolfssl || PACKAGE_libwolfsslcpu-crypto
```
This is the commit
https://github.com/openwrt/openwrt/commit/d08c9da43cf364712e947d5faa3ab84d995dd0ec

---

I was forgetting about this.  Thanks.